salt/modules/win_timezone.py

```diff
@@ -6,590 +6,387 @@
 
 # Import Python libs
 import logging
-import re
+import pytz
```

Comment:
This is not a hard dep for Salt, is it? If not, then this will cause a lot of tracebacks on non-Windows boxes which don't have pytz installed, unless we gate this in a try/except to catch the `ImportError`.

Also, we'd need to make sure that we add pytz to the freezer_includes or whatever we use now when making the Windows release.

What about the gist I posted in [this comment](https://github.com/saltstack/salt/issues/47559#issuecomment-388572450)? It seems like there was a way to get this to work using a different flag for tzutil. That would be the optimal solution as it would not require us to dep on another module.